Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -26,6 +26,44 @@ public MatchService(CCTContext context,  IAccountService accountService)
             _context = context;
             _accountService = accountService;
         }
+
+        public IEnumerable<LookupViewModel> GetMatchLookup(string type)
+        {
+            if (type == "status")
+            {
+                var res = _context.MatchStatus
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
+            if (type == "teamstatus")
+            {
+                var res = _context.MatchTeamStatus
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
+            if (type == "surface")
+            {
+                var res = _context.Surface
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
```

Comment:
Just a comment after the fact...this would make a great switch statement!

Reply:
forgot c# had that, I'll make it a change on my next set of changes